packages/ui/src/components/va-data-table/hooks/useSortable.ts

```diff
@@ -64,8 +65,8 @@ export const useSortable = (
       if (props.sortingOrder === undefined) {
         sortingOrderFallback.value = value
       }
-
-      emit('update:sortingOrder', value)
+      const columnName = sortedColumn.value
+      emit('update:sortingOrder', { value, columnName })
```

Comment:
In this example:
```ts
const sort = ref('desc')

v-model:sortingOrder="sort"
```
sort will be updated from 'desc' to { value: 'asc', columnName: 'user' }